qpc/cli.py

```diff
@@ -75,8 +103,13 @@ def __init__(self, name="cli", usage=None, shortdesc=None, description=None):
         self.shortdesc = shortdesc
         if shortdesc is not None and description is None:
             description = shortdesc
-        self.parser = ArgumentParser(usage=usage, description=description)
+        self.parser = argparse.ArgumentParser(usage=usage, description=description)
         self.parser.add_argument("--version", action="version", version=VERSION)
+        self.parser.add_argument(
+            "--version-git",
+            action=VersionGitAction,
+            help=argparse.SUPPRESS,
```

Comment:
This is the magic that hides the command from `--help`. Remove this arg, and it shows up normally.

Reply:
🪄